Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,40 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            
+            if (array == null || array.Length == 0)
+                return;
+
+            int[] newArray = new int[array.Length];
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                int position = 0;
+                for (int k = 0; k < array.Length; k++)
+                {
+                    if (array[i] > array[k])
+                    {
+                        position++;
+                    }
+                }
+                bool isSameNumberCheck = true;
+
+                while (isSameNumberCheck)
+                {
+                    if (array[i] == 0)
+                        isSameNumberCheck = false;
+                    else if (array[i] == newArray[position])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886613)